packages/captp/test/test-sync.js

```diff
@@ -0,0 +1,84 @@
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava';
+import { Far } from '@agoric/marshal';
+import { E, makeCapTP, makeLoopback } from '../lib/captp';
+import { nearGetApplySync } from '../lib/sync';
+
+function createFarBootstrap(exportAsSyncable) {
+  // Create a remotable that has a syncable return value.
+  return Far('test sync', {
+    getSyncable(n) {
+      const syncable = exportAsSyncable(
+        Far('getN', {
```

Comment:
Maybe this should be spelled `FarSync(iface, object)` or `Far(iface, object, {sync: true})`

Reply:
`FarSync` sounds reasonable.  I'll consider it!

Reply:
Settled on `makeTrapHandler`.